examples/data-objects/codemirror/src/codemirror.ts

```diff
@@ -223,8 +222,7 @@ export class CodeMirrorComponent
         /* Private */ context: IFluidDataStoreContext,
     ) {
         super();
-        this.url = context.id;
-        this.innerHandle = new FluidObjectHandle(this, this.url, runtime.objectsRoutingContext);
+        this.innerHandle = new FluidObjectHandle(this, "", runtime.objectsRoutingContext);
```

Comment:
The url gets changes to `""` here, but you use `"/"` when requesting the handle in other places.  Since they're equivalent (or at least when requesting the handle looks like), should the same value be used for an empty request for more consistency?  Or would this be problematic when requesting handles from not the default (e.g. `"/text"` from a change below)?  i.e. would `"text"` be wrong in that scenario?

Reply:
Passing here "/" will produce slightly different URL (with / at the end), but otherwise it will resolve the same way - that's because most of the parsing code would ignore trailing slash when parsing full URI.
Ideally, all of this code with manual parsing of request should go away. I have a prototype [here](https://github.com/microsoft/FluidFramework/compare/main...vladsud:Routing?expand=1) where I experiment with these things and slowing bringing them into main. Main idea - request routing and generation of handles should be one and the same thing, not two sub-systems that may be not in sync.

Yes, you are likely right that using "" when requesting likely is more readable, but request() implementations have to handle both "/" and "" simply because we currently do not have uniform way of how to parse URIs (i.e. there is no protocol in place that prior layer has to produce "" or "/" when processing and removing parts of URI).

Same for "text" vs. "/text" - both are valid and it's artifact of same thing - being careful and allowing prior parsing layer to keep or remove /.

I guess the summary - more work is needed here for all this code to be more readable, more uniform, and have less chances of making mistakes. One change at a time :)